extensions/io/org.eclipse.smarthome.io.javasound/src/main/java/org/eclipse/smarthome/io/javasound/internal/JavaSoundAudioSink.java

```diff
@@ -118,4 +136,7 @@ private void runVolumeCommand(Closure closure) {
         }
     }
 
+    private boolean isMacOS() {
+        return System.getProperty("osgi.os").equals("macosx");
```

Comment:
It would be great if it is not necessary at all.
But if we need to use it, we should do it using this one:
* https://wiki.searchtechnologies.com/javadoc/osgi/osgi-javadoc-v41/org/osgi/framework/Constants.html#FRAMEWORK_OS_NAME
* https://www.osgi.org/developer/specifications/reference/#ReferenceOS